pkgs/by-name/ol/olympus/unwrapped.nix

```diff
@@ -0,0 +1,106 @@
+{
+  lib,
+  fetchFromGitHub,
+  fetchzip,
+  buildDotnetModule,
+  dotnetCorePackages,
+  luajitPackages,
+  sqlite,
+  libarchive,
+  curl,
+  love,
+  xdg-utils,
+}:
+let
+  lua_cpath =
+    with luajitPackages;
+    lib.concatMapStringsSep ";" getLuaCPath [
+      (buildLuarocksPackage {
+        pname = "lsqlite3";
+        version = "0.9.6-1";
+        src = fetchzip {
+          url = "http://lua.sqlite.org/home/zip/lsqlite3_v096.zip";
+          hash = "sha256-Mq409A3X9/OS7IPI/KlULR6ZihqnYKk/mS/W/2yrGBg=";
+        };
+        buildInputs = [ sqlite.dev ];
+      })
+
+      lua-subprocess
+      nfd
+    ];
+
+  pname = "olympus";
+  phome = "$out/lib/${pname}";
+  # The following variables are to be updated by the update script.
+  version = "25.02.07.01";
+  buildId = "4624"; # IMPORTANT: This line is matched with regex in update.sh.
+  rev = "f4cd9dc973e68dc9b6c043941d5ab57f93b63ac4";
```

Comment:
Please run the update script to update the version. The latest version right now is 25.04.20.01.